lib/metrics/runtime.js

```diff
@@ -0,0 +1,62 @@
+'use strict'
+
+const v8 = require('v8')
+const loopDelay = require('event-loop-delay')
+
+class RuntimeCollector {
+  constructor () {
+    this.stats = {
+      'node.active_handles': 0,
+      'node.active_requests': 0,
+      'node.cpu.system': 0,
+      'node.cpu.user': 0,
+      'node.event_loop.delay': 0,
+      'node.event_loop.times': 0,
+      'node.heap.allocated': 0,
+      'node.heap.used': 0
+    }
+
+    this.lastCpu = process.cpuUsage()
```

Comment:
This was added in Node.js v6.1.0, and while I hope no one is running with Node.js 6.0.0, our tests do test against that version. So I think this line will fail.

Reply:
fixed